src/gluonts/model/deepar/_estimator.py

```diff
@@ -154,6 +155,16 @@ def __init__(
         ), "The value of `context_length` should be > 0"
         assert num_layers > 0, "The value of `num_layers` should be > 0"
         assert num_cells > 0, "The value of `num_cells` should be > 0"
+        supported_dropoutcell_types = [
+            "ZoneoutCell",
+            "RNNZoneoutCell",
+            "VariationalDropoutCell",
+            "VariationalZoneoutCell",
+        ]
+        assert dropoutcell_type in supported_dropoutcell_types, (
+            "`dropoutcell_type` should be one of %s"
+            % (str(supported_dropoutcell_types))
```

Comment:
Better to use f-strings:

```suggestion
            f"`dropoutcell_type` should be one of {supported_dropoutcell_types}"
```